lib/streamlit/navigation/page.py

```diff
@@ -141,6 +158,10 @@ def title(self) -> str:
     def icon(self) -> str:
         return self._icon
 
+    @property
+    def url_path(self) -> str:
+        return "" if self._default else self._url_path
```

Comment:
nit: perhaps it makes sense to have a short comment to explain why we return `""` for default pages. Or maybe better is to add it to the docstring [here](https://github.com/streamlit/streamlit/pull/8778/files#diff-05ad866665a65e2f749eee48ae5169a66433f51ff99064a0af1791cb7fb3f3b7R75-R77) as it sounds like `url_path` will be ignored when `default=True`